kuma/static/styles/zone-ten.scss

```diff
@@ -1,18 +0,0 @@
-@import 'includes/vars';
-@import 'includes/mixins';
-
-/*
-MDN at 10 zone
-********************************************************************** */
-
-
-/*
-imports
-====================================================================== */
-@import 'zones/base';
-@import 'zones/components/chapters';
-@import 'zones/components/learnmore';
-@import 'zones/components/promo10';
-@import 'zones/components/quote';
-
-@include zone-image('ten.svg', 66.7, 55.26);
```

Comment:
This is incredibly helpful. The only one here that bothers me is the change to archived docs. It is useful having a very ugly style for those docs, since we don't really have any other way to differentiate them. We could probably live without this style, but it would be worth having if it's not to difficult. Unfortunately the style doesn't seem to be applied using a class, so I'm not sure how we could preserve it using "common CSS".

I don't think "MDN at ten" is very high priority content - it was essentially a marketing exercise three years ago.

Reply:
This also reminds me that we need a replacement sidebar for the Games zone: https://github.com/mdn/kumascript/issues/727.

Reply:
I find it useful to see when different engines were implemented. Kuma has actually been one of the more long-lived ones. Which may mean we're due for the next engine update...

I think the content could be refactored to work without the custom styles, and maybe redirect the "Contributing to MDN" page to http://localhost:8000/en-US/docs/MDN/Getting_started. This could be tracked in a content bug.